scripts/bootstrap

```diff
@@ -54,32 +54,39 @@ success_msg() {
   colored_msg "${SUCCESS_COLOR}" "SUCCESS" "${1}"
 }
 
+TGZ_PATH=""
+
 # Displays program name
 msg "PROGRAM: ${PROGRAM}"
 
-# The real bootstrap script starts 20 lines below ;-)
+# options may be followed by one colon to indicate they have a required argument
+if ! options=$(getopt -o dsb -l -t deploy,ci,tgz_path: -- "$@")
+then
+    # something went wrong, getopt will put out an error message for us
+    exit 1
+fi
+
+# The real bootstrap script starts with app installation below ;-)
 flags=()
 deploy=false
 cmd="pipenv sync"
 ci=false
-while test $# -gt 0; do
-    case "$1" in
-        --deploy)
-            # We are probably inside travis, check for env variables
-            deploy=true
-            shift
-            ;;
-        --ci)
-            # We are probably inside travis, check for env variables
-            ci=true
-            shift
-            ;;
-        *)
-            # Use the flags that are passed to the script directly to install
-            flags+=("$1")
-            shift
-            ;;
-    esac
+while test $# -gt 0;
```

Comment:
I think `;` is not needed if you go to the line after. It was only if you put `do` after that.